CHANGELOG.en-US.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 2.24.7 (2022-01-26)
+
+### Fixes
+
+- Update translation for `jaJP` locale.
+
```

Comment:
1.changelog location is error, please see CONTRIBUTE.md
2. use i18n replace fixes
![image](https://user-images.githubusercontent.com/34439652/151105231-0bab79ae-ef62-4063-b2bd-0296ccfcd451.png)